node/src/chain_spec.rs

```diff
@@ -243,7 +244,9 @@ pub fn finney_config() -> Result<ChainSpec, String> {
 			)
 		},
 		// Bootnodes
-		vec![],
+		vec![
+			"/ip4/13.58.175.193/tcp/30333/p2p/12D3KooWEXY393DjTvBip3DnZeJkSAeZrg6edxqu45ywHzzQS2BY".parse().unwrap()
```

Comment:
can we place this IP somewhere that makes it clear for us that is something that could change? Some config file or constants files?